pydantic/types.py

```diff
@@ -230,8 +231,31 @@ def confrozenset(
 
 
 def conlist(
-    item_type: type[AnyItemType], *, min_length: int | None = None, max_length: int | None = None
+    item_type: type[AnyItemType],
+    *,
+    min_length: int | None = None,
+    max_length: int | None = None,
+    unique_items: bool | None = None,
 ) -> type[list[AnyItemType]]:
+    """A wrapper around typing.List that adds validation.
+
+    Args:
+        item_type (type[AnyItemType]): The type of the items in the list.
+        min_length (int | None, optional): The minimum length of the list. Defaults to None.
+        max_length (int | None, optional): The maximum length of the list. Defaults to None.
+        unique_items (bool | None, optional): Whether the items in the list must be unique. Defaults to None.
+
+    Returns:
+        type[list[AnyItemType]]: The wrapped list type.
+    """
```

Comment:
Do you prefer all docstrings in the same PR, or can I take the opportunity here?